src/types/lang.rs

```diff
@@ -0,0 +1,148 @@
+use std::str::FromStr;
+
+use strum::{Display, EnumString};
+
+#[cfg(feature = "unic-langid")]
+use unic_langid::{langid, LanguageIdentifier};
+
+use super::{Type, TypeError, TypeErrorKind};
+use crate::{Chunk, Chunks, ChunksExt, ChunksRef, Spanned};
```

Comment:
I think these can be merged (some formatting needed)
```suggestion
use crate::{Chunk, Chunks, ChunksExt, ChunksRef, Spanned, Type, TypeError, TypeErrorKind};
```